src/github.com/getlantern/flashlight/flashlight.go

```diff
@@ -66,54 +66,63 @@ func Run(httpProxyAddr string,
 	socksProxyAddr string,
 	configDir string,
 	stickyConfig bool,
-	proxyAll func() bool,
+	proxyAll bool,
 	flagsAsMap map[string]interface{},
 	beforeStart func(cfg *config.Config) bool,
 	afterStart func(cfg *config.Config),
 	onConfigUpdate func(cfg *config.Config),
-	onError func(err error)) error {
+	onError func(err error)) (proxyAllUpdater func(bool), err error) {
```

Comment:
It is ugly to add this function as return value, but limited by current code structure, it's a quick and dirty way to fix.